containers.h

```diff
@@ -53,6 +53,7 @@ typedef std::map<std::string, packToken> TokenMap_t;
 struct MapData_t {
   TokenMap_t map;
   TokenMap* parent;
+  MapData_t();
```

Comment:
This constructor is not used anywhere if I am not mistaken. Maybe we should not add it in this PR.

Reply:
If I remember correctly my visual studio compiler was enraged about this missing constructor. I can manage that locally, in my own space, although it is a bit of trouble for me.

Reply:
I see, it that's the case then it would probably affect other people as well. Better to keep it then =].

I will mark this one as resolved.